hardware/OpenWeatherMap.cpp

```diff
@@ -145,6 +145,8 @@ void COpenWeatherMap::GetMeterDetails()
 	{
 		bool bret;
 		std::string szURL = sURL.str();
+		HTTPClient::SetConnectionTimeout(5);
+		HTTPClient::SetTimeout(5);
```

Comment:
Why do you need to set this type of timeout ? You know it's not a connection timeout ?
Whats wrong with the default timeout of 10 seconds ? I suppose nothing, and if you have a fast internet connection it will take a sec ?